### ISSUE TITLE
Search results page is missing a language tag

### ISSUE THREAD

**Issue by Phanoix** [gctools-outilsgc/gcconnex#20]
Search results page is missing a language tag, which should most likely simply declare the interface language, as the found user content language can be in any language, or even a mix.
